pwndbg/commands/heap.py

```diff
@@ -1546,3 +1547,74 @@ def heap_config(filter_pattern: str) -> None:
             "Some config values (e.g. main_arena) will be used only when resolve-heap-via-heuristic is `auto` or `force`"
         )
     )
+
+
+# Jemalloc
+
+parser = argparse.ArgumentParser(
+    description="Returns extent information for pointer address allocated by jemalloc"
```

Comment:
> Memory is conceptually broken into extents. Extents are always aligned to multiples of the page size. This alignment makes it possible to find metadata for user objects quickly. User objects are broken into two categories according to size: small and large. Contiguous small objects comprise a slab, which resides within a single extent, whereas large objects each have their own extents backing them.

hmm... wonder if we can at least add a comment on those extents somewhere or inform the user about them, idk...

Reply:
Added information on extent in comments